src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -1452,7 +1452,7 @@ void TwitchMessageBuilder::modsOrVipsSystemMessage(QString prefix,
     builder->emplace<TimestampElement>();
     builder->message().flags.set(MessageFlag::System);
     builder->message().flags.set(MessageFlag::DoNotTriggerNotification);
-    builder->emplace<TextElement>(prefix, MessageElementFlag::Text,
+    builder->emplace<TextElement>(prefix + " ", MessageElementFlag::Text,
```

Comment:
```suggestion
    builder->emplace<TextElement>(prefix, MessageElementFlag::Text,
```

![image](https://user-images.githubusercontent.com/41973452/141472706-a44a8735-73a8-4ab1-bea8-90ac54724d58.png) - current
vs
![image](https://user-images.githubusercontent.com/41973452/141472558-60bdea83-f71c-40c7-a202-37a53432d11d.png) - change